pkg/api/types.go

```diff
@@ -2154,3 +2154,38 @@ type RangeAllocation struct {
 	// a single allocated address (the fifth bit on CIDR 10.0.0.0/8 is 10.0.0.4).
 	Data []byte `json:"data"`
 }
+
+// LockSpec is the specification of a resource lock.
+type LockSpec struct {
+	// Who currently has the lock.
+	HeldBy string `json:"heldBy"`
+	// How long the lock is valid.
+	LeaseSeconds uint64 `json:"leaseSeconds"`
```

Comment:
Is there some undocumented convention that unsigned ints shouldn't be used?  For something like a timeout it makes no sense to allow a negative number.  Are you saying unsigned ints aren't allowed in the api at all?  There doesn't appear to be a standard (see int, int64).

I really don't care either way, but it would be nice to know why an uint is a problem.